portal/views.py

```diff
@@ -25,9 +25,8 @@ def get_public_apps(request, id=0, get_all=True, collection=None):
         )  # noqa: F841 local var assigned but never used
         logger.info(len(projects))
     except Exception:
-        logger.debug(
-            "User not logged in.",
-        )
+        # logger.debug("User not logged in.")
+        pass
```

Comment:
hmm this strikes me as a bit odd. Should we remove the try completely instead? Or what is the motivation for doing this?

Reply:
Perhaps. We have a refactor TODO on this function that we can revisit during that epic.